analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/Extensions/IOperationExtensions.cs

```diff
@@ -63,9 +63,15 @@ internal static class IOperationExtensions
         internal static IInvocationOperationWrapper ToInvocation(this IOperation operation) =>
             IInvocationOperationWrapper.FromOperation(operation);
 
+        internal static IFieldReferenceOperationWrapper ToFieldReference(this IOperation operation) =>
```

Comment:
I see `Implement S2259 FN for Nullables` in the commit history. It would be nice to have "Squash" in the name so I know that I don't need to review that part. As it adds noise on GitHub UI (that's fine as long as I know why).

Also, this is not mergeable, rebase will be needed => please convert it to draft.